site-packages/integralstor/cifs.py

```diff
@@ -188,31 +513,14 @@ def generate_smb_conf():
 
 def reload_configuration():
     try:
-        use_salt, err = config.use_salt()
-        if err:
-            raise Exception(err)
-        if use_salt:
-            import salt.client
-            errors = ''
-            client = salt.client.LocalClient()
-            r1 = client.cmd('*', 'cmd.run_all',
-                            ['smbcontrol all reload-config'])
-            if r1:
-                for node, ret in r1.items():
-                    # print ret
-                    if ret["retcode"] != 0:
-                        errors += "Error reloading samba on node %s " % node
-            if errors:
-                raise Exception(errors)
-        else:
-            cmd_to_run = 'smbcontrol all reload-config'
-            lines, err = command.get_command_output(cmd_to_run)
-            if err:
-                raise Exception(err)
-            ret, err = services_management.update_service_status(
-                'winbind', 'restart')
-            if err:
-                raise Exception(err)
+        cmd_to_run = 'smbcontrol all reload-config'
+        lines, err = command.get_command_output(cmd_to_run)
+        if err:
+            raise Exception(err)
+        ret, err = services_management.update_service_status(
+            'winbind', 'restart')
```

Comment:
Should we include 'smb' to restart as well? Just as a fail safe to smbcontrol reload-config.

Reply:
I'll add that with the next PR for the other files..